docs/feature_combo.md

```diff
@@ -67,31 +86,100 @@ void process_combo_event(uint8_t combo_index, bool pressed) {
         tap_code16(LCTL(KC_V));
       }
       break;
+    case SD_LAYER:
+      /* until more advanced keycodes are supported, layer changing combos need to be handled manually. */
+      if (pressed) {
+        layer_on(_LAYER);
+      } else {
+        layer_off(_LAYER);
+      }
+      break;
   }
 }
 ```
 
-This will send Ctrl+C if you hit Z and C, and Ctrl+V if you hit X and V.  But you could change this to do stuff like change layers, play sounds, or change settings.
-
-## Additional Configuration
-
-If you're using long combos, or even longer combos, you may run into issues with this, as the structure may not be large enough to accommodate what you're doing.
-
-In this case, you can add either `#define EXTRA_LONG_COMBOS` or `#define EXTRA_EXTRA_LONG_COMBOS` in your `config.h` file.
+This will send Ctrl+C if you hit Z and C, Ctrl+V if you hit X and V, and changel layer to `_LAYER` when you hold down S and D.  But you could change this to do stuff like play sounds or change settings.
 
-You may also be able to enable action keys by defining `COMBO_ALLOW_ACTION_KEYS`.
+# Advanced Configuration
+These configuration settings can be set in your keymaps `config.h`. 
 
-## Keycodes 
-
-You can enable, disable and toggle the Combo feature on the fly.  This is useful if you need to disable them temporarily, such as for a game. 
+#### Keycodes 
+You can enable, disable and toggle the Combo feature on the fly.  This is useful if you need to disable them temporarily, such as for a game. The following keycodes are avalible for use in your `keymap.c`
 
 |Keycode   |Description                      |
 |----------|---------------------------------|
 |`CMB_ON`  |Turns on Combo feature           |
 |`CMB_OFF` |Turns off Combo feature          |
 |`CMB_TOG` |Toggles Combo feature on and off |
 
-## User callbacks
+#### Combo Term
+By default, the timeout for the Combos to be recognized is set to 50ms. This can be changed if accidental combo misfires are happening or if you're having difficulties pressing keys at the same time. For instance `#define COMBO_TERM 40` would set the time out period for combos to 40ms.
+
+#### Long Combos
+If you're using long combos or longer combos, you may run into issues with this, as the structure may not be large enough to accommodate what you're doing. In this case, you can configure the size of the data structure used. Be aware, larger combo sizes will increase memory usage!
```

Comment:
« long combos or longer combos » : it feels like some words are missing. I suggest keeping the same wording as on the actual (i.e. the one in `qmk:master`) combo docs page.
« If you're using long combos or longer combos, » → « If you're using long combos, or even longer combos, »